docs/whats-new.asciidoc

```diff
@@ -4,165 +4,150 @@
+Each of the Explore detail pages ({security-guide}/hosts-overview.html#host-details-page[host details], {security-guide}/network-page-overview.html#ip-details-page[network details], and {security-guide}/users-page.html#_user_details_page[user details]) now displays alert metrics that show the total number of alerts by severity, rule, and status.  
 
-[discrete]
-== Cases enhancements
-
-*New Webhook - Case Management case connector*
+[role="screenshot"]
+image::whats-new/images/8.5/alert-counts.png[Alert count metrics]
 
-The Webhook - Case Management connector allows you to build a custom connector for any third-party case/ticket management system. This offers more flexibility when deciding what third-party case/ticket management system you want to send cases and case updates to.
+*Visual event analyzer enhancements* 
 
-*New sub-feature privilege for cases*
+* If you have a http://elastic.co/pricing[Platinum subscription] or higher, you can now examine alerts associated with the event by default. 
 
-The *Delete cases and comments* sub-feature privilege determines whether a user can delete cases and comments. Users with current `All` access to cases are automatically granted the delete privilege upon upgrading to 8.4. However, users with current `read` access to cases are not automatically granted the delete privilege upon upgrading to 8.4. An admin can modify these user privileges.
+* The visual event analyzer automatically displays the entire process tree if you select a time range that doesn't have any process events. You'll also receive a message that your time range is too narrow. 
 
-[role="screenshot"]
-image::whats-new/images/8.4/cases-privs.png[Cases privileges]
 
-[discrete]
-== Endpoint enhancements
+*Alert details flyout enhancements* 
 
-*New credential hardening protection*
+The following enhancements have been made to the alert details flyout. To learn more about analyzing detection alerts, refer to {security-guide}/view-alert-details.html[View alert details]. 
 
-You can now configure {security-guide}/configure-endpoint-integration-policy.html#attack-surface-reduction[credential hardening protection] in an integration policy. Credential hardening prevents attackers from stealing credentials stored in Windows system process memory. Turn on the toggle to remove any overly permissive access rights that aren’t required for standard interaction with the Local Security Authority Subsystem Service (LSASS).
+* If you have a http://elastic.co/pricing[Platinum subscription] or higher, these details are included in the *Insights* section:
 
+** *Event renderer added to the Overview tab* - The alert rendered view displays event details, such as the alert reason statement, file paths, or process arguments, to provide context for the alert. You can take action (such as Add to Timeline) on individual fields in the statement. 
```

Comment:
```suggestion

```

Reply:
Moving this out since it's a change to the Overview tab, not the Insights section. :)